timeseries/src/autogluon/timeseries/models/chronos/pipeline/utils.py

```diff
@@ -317,7 +317,7 @@ def on_train_begin(self, args, state, control, **kwargs):
     def on_step_end(self, args, state, control, **kwargs):
         elapsed_time = time.monotonic() - self.start_time
         if elapsed_time > self.time_limit:
-            raise TimeLimitExceeded
+            control.should_training_stop = True
```

Comment:
log out a warning? fine-tuning had to early stop, i.e., the max_steps the user wanted could not be observed..